README.md

```diff
@@ -76,16 +76,27 @@ They have a short header and then the page content in the form:
 
 /---
 layout: default
-title: Getting Started with OMERO.insight Version 5.0.3
+title: Getting Started with OMERO.insight
```

Comment:
This text is formatted without line breaks in markdown, I think you probably need to put it in a code box if you want to preserve the line breaks and spacing.